src/main/java/io/fabric8/vertx/maven/plugin/mojos/AbstractRunMojo.java

```diff
@@ -423,14 +422,22 @@ private boolean isYaml(String configFile) {
 
         @Override
         public void call(String kind, Path path) {
```

Comment:
If you remove it, you could use a `Callable` directly without having to invent a new interface. The logging of the kind happens in the IncrementalBuilder anyway.